katana/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mototdd.katana">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
```

Comment:
Are we supporting RTL?

Reply:
This appears to be an artifact of the project generation and I did not intentionally add it.

Lets discuss this at our next meeting. My experience tells me that the small cost of adding international/localizing the UI may give this s larger audience plus increased learnings and finds. 

- [ ] Discuss RTL and localization.